decksite/controllers/api.py

```diff
@@ -143,6 +157,42 @@ def decks_api() -> Response:
     resp.set_cookie('page_size', str(page_size))
     return resp
 
+@APP.api.route('/updated/')
```

Comment:
I thought we were going with /decks/updated? We really want a noun in there.

Reply:
oops